packages/features/eventtypes/components/tabs/limits/EventLimitsTab.tsx

```diff
@@ -23,7 +24,40 @@ import { Button, DateRangePicker, InputField, Label, Select, SettingsToggle, Tex
 
 type IPeriodType = (typeof PeriodType)[keyof typeof PeriodType];
 
-export type EventLimitsTabProps = Pick<EventTypeSetupProps, "eventType">;
+export type EventLimitsTabCustomClassNames = {
+  bufferAndNoticeSection?: {
+    container?: string;
+    beforeBufferSelect?: SelectClassNames;
```

Comment:
Question - what does `innerClassNames` mean within `SelectClassNames` ? These are classnames already existing in the code?

I am asking because if we have before buffer select:
```"beforeBufferSelect": {
        "container": "mb-4",
        "label": "text-purple-700",
        "select": "border border-purple-300 rounded-md text-purple-900 bg-white",
        "innerClassNames": {
          "input": "text-purple-900",
          "option": "text-purple-700 hover:bg-purple-200",
          "control": "border-purple-300",
          "singleValue": "text-purple-900",
          "valueContainer": "bg-purple-100",
          "menu": "border border-purple-200 rounded-md",
          "menuList": "bg-white"
        }
      },
```

we have `select` key and then `innerClassNames`. I wonder if we should rename `select` to `container` and `innerClassNames` to `select` and then move `container` within the `select` so we would have:
```
"beforeBufferSelect": {
        "container": "mb-4",
        "label": "text-purple-700",
        "select": {
          "container": "border border-purple-300 rounded-md text-purple-900 bg-white",
          "input": "text-purple-900",
          "option": "text-purple-700 hover:bg-purple-200",
          "control": "border-purple-300",
          "singleValue": "text-purple-900",
          "valueContainer": "bg-purple-100",
          "menu": "border border-purple-200 rounded-md",
          "menuList": "bg-white"
        }
      },
```

Reply:
and the same for others

Reply:
https://github.com/calcom/cal.com/blob/48c0676f411e97cbbe973607004af2e2ac8ff114/packages/ui/components/form/select/Select.tsx#L27
yes -- these are classnames we already have in our ui/select component